index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://sad-haibt-42112c.netlify.app/styles/main.css"
+    />
+    <link rel="stylesheet" href="/styles/signup.css" />
+    <script
+      src="https://kit.fontawesome.com/43943f2889.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Sign Up | Sanguine Store</title>
+  </head>
+  <body>
+    <nav class="navigation">
+      <div class="nav-logo">SANGUINE store</div>
+      <label for="search" class="text-size-sm">
+        <input
+          type="search"
+          class="search-box padding-md border-rad-md"
+          name="search"
+          id="search"
+          placeholder="What are you looking for?"
+        />
+        <a href="" class="options text-dec-none">Search</a>
+      </label>
+      <ul class="list-style-none list-inline">
+        <li>
+          <a class="options link" href="">profile</a>
+        </li>
+        <li>
+          <a class="options link" href="">wishlist</a>
+        </li>
+        <li>
+          <a class="options link" href="">cart</a>
+        </li>
+      </ul>
+    </nav>
+    <section class="signup-form">
+      <div class="input-container">
+        <h3 class="h3 text-wt-bold">Sign Up</h3>
+        <label for="input-text">E-mail</label>
+        <input
+          type="email"
+          name="input-text"
+          class="txt-box"
+          id="input-text"
+          placeholder="E-mail"
+          required
+        />
+        <label for="input-text">First name</label>
+        <input
+          type="email"
+          name="input-text"
+          class="txt-box"
+          id="input-text"
+          placeholder="First name"
+          required
+        />
+
+        <label for="input-text">Last name</label>
+        <input
+          type="email"
+          name="input-text"
+          class="txt-box"
+          id="input-text"
+          placeholder="Last name"
+          required
+        />
+
+        <label for="input-error">Password</label>
+        <input
+          type="pwd"
+          name="input-error"
+          class="txt-box error"
+          id="input-error"
+          placeholder="Passwords don't match"
+        />
+
+        <label for="input-error">Confirm password</label>
+        <input
+          type="pwd"
+          name="input-error"
+          class="txt-box error"
+          id="input-error"
+          placeholder="Passwords don't match"
+        />
+
+        <button class="btn btn-primary" type="submit">Sign Up</button>
+        <span class="text-size-sm"
+          >Already have an account?
```

Comment:
It might look better if this is centered

Reply:
Sure, will do this.